jujugui/static/gui/src/app/components/header-search/test-header-search.js

```diff
@@ -66,27 +66,6 @@ describe('HeaderSearch', function() {
       </span>);
   });
 
-  it('changes state when the close button is clicked', function() {
```

Comment:
Looks to me like the state is still changed when the close button is clicked. Is this close button case now tested elsewhere?